packages/client/mutations/toasts/mapResponseMentionedToToast.ts

```diff
@@ -0,0 +1,46 @@
+import graphql from 'babel-plugin-relay/macro'
```

Comment:
Future notifications will only need a file like this + a couple lines in `popNotificationToast` to implement the toast - no need to implement an `onNext` handler and get into the internals of the notification subscription, this should Just Work ™️